.github/workflows/publish.yml

```diff
@@ -40,8 +40,19 @@ jobs:
       - name: Install mdbook
         uses: ./.github/workflows/install-mdbook
 
+      - name: Install mdbook-pandoc
+        run: cargo install mdbook-pandoc
+      - name: Install pandoc
+        run: |
+          curl -LsSf https://github.com/jgm/pandoc/releases/download/3.1.9/pandoc-3.1.9-linux-amd64.tar.gz | tar zxf -
+          echo "$PWD/pandoc-3.1.9/bin" >> $GITHUB_PATH
+      - name: Install mdbook-pandoc dependencies
+        run: sudo apt-get install -y texlive texlive-luatex librsvg2-bin
```

Comment:
I think we could put all these steps into a single `Install mdbook-pandoc` step. Just to group things a bit.